.claude/agents/test-runner.md

```
@@ -0,0 +1,123 @@
+## 実行フロー
+
+1. `reports/tests/` ディレクトリを作成（存在しない場合）
+2. `npm test` を実行
+3. 結果をレポートファイルに出力
+4. 失敗があれば詳細情報を含めてメインエージェントに報告
+
+## レポート出力ルール
+
+### フォルダ構造
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)